save-demo/src/main/resources/application.properties

```diff
@@ -10,7 +10,8 @@ spring.jpa.properties.hibernate.order_inserts=true
 spring.jpa.properties.hibernate.order_updates=true
 spring.liquibase.enabled=false
 logging.level.org.hibernate.engine.internal.StatisticalLoggingSessionEventListener=WARN
-demo.backend=http://backend:5800/internal
+demo.backend-url=http://backend:5800/internal
+demo.agent-config.demo-url = http://demo:5421
```

Comment:
as I remember we already discussed mapping of ports (5421) and that port which ends on 1 is for management in k8s. do we still want to use it?

as I remember, @petertrr suggested to migrate to :80 for webservice and a single dedicated port for management for all services

Reply:
Migration to ':80' port for Kubernetes run (`kubernetes` profile) seems to be a good idea. Probably we need to do it, but in a separate PR.